guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -0,0 +1,365 @@
+
+[[Lifecycle_Environment]]
+Lifecycle Environment:: A container for content view versions consumed by the content hosts.
+A Lifecycle Environment represents a step in the lifecycle environment path.
+Content moves through lifecycle environments by publishing and promoting content views.
+
+[[Lifecycle_Environment_Path]]
+Lifecycle Environment Path:: A sequence of lifecycle environments through which the content views are promoted.
+You can promote a content view through a typical promotion path; for example, from development to test to production.
+
+[[Manifest]]
+Manifest (Red{nbsp}Hat Subscription Manifest):: A mechanism for transferring subscriptions from the Red{nbsp}Hat Customer Portal to {ProjectName}.
+Do not confuse with xref:Puppet_Manifest[Puppet Manifest].
+
+[[Migrating_{project-context}]]
+Migrating {Project}:: The process of moving an existing {Project} installation to a new instance.
+
+[[OpenSCAP]]
+OpenSCAP:: A project implementing security compliance auditing according to the Security Content Automation Protocol (SCAP).
+OpenSCAP is integrated in {Project} to provide compliance auditing for hosts.
+
+ifdef::orcharhino[]
+[[{smart-proxy-context}]]
+{SmartProxy}:: An additional server that can be used in a {ProjectName} deployment to facilitate content federation and distribution (act as a Pulp mirror), and to run other localized services (Puppet server, *DHCP*, *DNS*, *TFTP*, and more).
+{SmartProxies} are useful for {Project} deployment across various geographical locations.
+In upstream Foreman terminology, {SmartProxy} is referred to as Smart Proxy.
+endif::[]
+
+[[Organization]]
+Organization:: An isolated collection of systems, content, and other functionality within a {Project} deployment.
+
+[[Parameter]]
+Parameter:: Defines the behavior of {ProjectName} components during provisioning.
+Depending on the parameter scope, we distinguish between global, domain, host group, and host parameters.
+Depending on the parameter complexity, we distinguish between simple parameters (key-value pair) and smart parameters (conditional arguments, validation, overrides).
+
+[[Parametrized_Class]]
+Parametrized Class (Smart Class Parameter):: A parameter created by importing a class from Puppet server.
+
+[[Permission]]
+Permission:: Defines an action related to a selected part of {Project} infrastructure (resource type).
+Each resource type is associated with a set of permissions, for example the *Architecture* resource type has the following permissions: *view_architectures*, *create_architectures*, *edit_architectures*, and *destroy_architectures*.
+You can group permissions into roles and associate them with users or user groups.
+
+[[Product]]
+Product:: A collection of content repositories.
+Products are either provided by Red{nbsp}Hat CDN or created by the {Project} administrator to group custom repositories.
+
+[[Promote]]
+Promote (a Content View):: The act of moving a content view from one lifecycle environment to another.
+
```

Comment:
Another good place to point out that this should IMO be explained directly in the respective module(s). For example, by xrefing the assembly or procedure about promoting CVs.

Reply:
Added